docs/components/gltf-model.md

```diff
@@ -95,6 +95,14 @@ rendering.
 <a-entity gltf-model="url(/path/to/tree.gltf)"></a-entity>
 ```
 
+## Using animations
+
+If you want to use the animations from your glTF model, you can use the [animation-mixer](https://github.com/donmccurdy/aframe-extras/tree/master/src/loaders#animation) component from [aframe-extras](https://github.com/donmccurdy/aframe-extras). By default the first animation is played in a loop. 
```

Comment:
Oops, my docs were incorrect — by default _all_ animations will play in a loop.

Reply:
hah, so two birds with one stone!